src/main/java/com/jcraft/jsch/OpenSSHConfig.java

```diff
@@ -143,6 +151,15 @@ public Config getConfig(String host) {
     return new MyConfig(host);
   }
 
+  /**
+   * Returns mapping of jsch config property names to OpenSSH property names.
+   *
+   * @return map
+   */
+  public static Hashtable<String, String> getKeymap() {
```

Comment:
Do we want to mark this as package private so that it doesn't become part of the public facing API?